src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,132 @@
+.menu {
+  background-color: $green;
+
+  &__content {
+    box-sizing: border-box;
+    padding: 27px 20px;
+  }
+
+  &__list {
+    box-sizing: border-box;
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+
+    border-bottom: 1px solid #fff;
+    padding: 0 0 24px;
+    margin: 0;
+
+    @include onDesktop {
+      border-right: 1px solid #fff;
+      border-bottom: none;
+    }
+
+    @include onTablet {
+      border-right: 1px solid #fff;
+      border-bottom: none;
+    }
+  }
+
+  &__link {
+    width: max-content;
+
+    display: block;
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 16px;
+
+    color: #fff;
+    border-bottom: 1px solid transparent;
+
+    text-decoration: none;
+    padding-top: 24px;
+
+    transition: border-color 0.3s;
+
+    &:hover {
+      border-color: #fff;
+    }
+
+    @include onTablet {
+      font-size: 16px;
+      line-height: 16px;
+    }
+
+    @include onDesktop {
+      font-size: 22px;
+      line-height: 22px;
+    }
+
+    &--top {
+      padding: 0;
+    }
+  }
+
+  &__title {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    color: #fff;
+    padding-bottom: 20px;
+    margin: 0;
+  }
+
+  &__schedue {
+    padding-bottom: 16px;
+
+    &--days {
+      font-family: "IBM Plex Sans", sans-serif;
```

Comment:
You should use the font-family defined in a global variable or mixin to keep consistency with other elements.